src/main/java/com/fortify/plugin/jenkins/steps/FortifySCAStep.java

```diff
@@ -86,31 +84,31 @@ public void setLogFile(String logFile) {
 
 	protected String getSourceAnalyzerExecutable(Run<?, ?> build, FilePath workspace, Launcher launcher,
 			TaskListener listener) throws InterruptedException, IOException {
-		return getExecutable("sourceanalyzer" + (launcher.isUnix() ? "" : ".exe"), true, build, workspace,
-				launcher, listener, null);
+		return getExecutable("sourceanalyzer" + (launcher.isUnix() ? "" : ".exe"), build, workspace,
+			listener, "FORTIFY_HOME");
 	}
 
 	protected String getMavenExecutable(Run<?, ?> build, FilePath workspace, Launcher launcher, TaskListener listener)
 			throws InterruptedException, IOException {
-		return getExecutable("mvn" + (launcher.isUnix() ? "" : ".cmd"), false, build, workspace, launcher,
-				listener, null);
+		return getExecutable("mvn" + (launcher.isUnix() ? "" : ".cmd"), build, workspace,
```

Comment:
Question:
1. What versions of maven does Jenkins plugin support in terms of system requirement? Maven uses 'bat' instead of 'cmd' till 3.2.5.
2. Is 'MAVEN_HOME' required by the Jenkins plugin or Maven's environment variable? According to this [stackoverflow article](https://stackoverflow.com/questions/17136324/what-is-the-difference-between-m2-home-and-maven-home#:~:text=2%20Answers&text=MAVEN_HOME%20is%20for%20Maven%201,both%20on%20the%20same%20machine.), MAVEN_HOME is for Maven 1 and M2_HOME is for Maven 2 and later.

Reply:
1. I don't think there are the restrictions for that as Maven is set manually through setting maven home variable and it can be any version of maven that user has installed and wants to use. I presume an additional check for '.bat' is needed in order to overcome the issue. Thanks for recommendation.
2. It's a good spot, thanks, I have added the fix.